################################################################################
Concuerror started with options:
  [{target,{after_test_2,after_test_2,[]}},
   {symbolic,true},
   {verbose,0},
   {timeout,1000},
   {files,["/home/stavros/git/Concuerror/testsuite/suites/dpor/src/after_test_2.erl"]},
   {'after-timeout',infinite},
   {bound,2},
   {distributed,true},
   {'light-dpor',false}]
################################################################################
Erroneous interleaving 1:
* Blocked at a 'receive' (when all other processes have exited):
    P in after_test_2.erl line 21
--------------------------------------------------------------------------------
Interleaving info:
   1: P: P.1 = erlang:spawn(erlang, apply, [#Fun<after_test_2.0.15042273>,[]])
    in erlang.erl line 2329
   2: P: receive timeout expired after 0 ms
    in after_test_2.erl line 11
   3: P: receive timeout expired after 0 ms
    in after_test_2.erl line 16
   4: P.1: one = P ! one
    in after_test_2.erl line 10
   5: Message (one) from P.1 reaches P
   6: P.1: exits normally
################################################################################
Erroneous interleaving 2:
* Blocked at a 'receive' (when all other processes have exited):
    P in after_test_2.erl line 21
--------------------------------------------------------------------------------
Interleaving info:
   1: P: P.1 = erlang:spawn(erlang, apply, [#Fun<after_test_2.0.15042273>,[]])
    in erlang.erl line 2329
   2: P: receive timeout expired after 0 ms
    in after_test_2.erl line 11
   3: P.1: one = P ! one
    in after_test_2.erl line 10
   4: P.1: exits normally
   5: Message (one) from P.1 reaches P
   6: P: receives message (one)
    in after_test_2.erl line 16
################################################################################
Done! (Exit status: ok)
  Summary: 2 errors, 2/2 interleavings explored
